about.html

```diff
@@ -0,0 +1,183 @@
+        prizes. <br>
+
+        Connect with a dynamic community of like-minded tech enthusiasts, forge new friendships, and collaborate on
+        fascinating
+        projects that push the boundaries of possibility. <br>
+      </span>
+    </p>
+    <p class="hero-contact">Please contact us by Email for any further questions about TechSplendor Symposium 2023!
+      <a href="mailto: tssymposium@tsuganda.org">tssymposium@tsuganda.org</a>
+    </p>
+  </section>
+
+  <main class="about-main">
+    <section class="section-1">
+      <h2>TechSplendor Symposium 2023 Logo</h2>
+      <hr>
+      <p>
+        The logo of TechSplendor Symposium 2023 was decided through the logo competition from 8,
+        June to 7, July.
+        <span>After online public voting and Selection Committee’s judging process, the logo from BestDesigners won the
+          prize.
+          After consultation with BestDesigners, we optimized the prize-winning-logo for this year’s symposium as
+          follows:</span>
+      </p>
+      <hr class="follows">
+
+      <img src="./images/tech-splendor.png" alt="Project logo">
+    </section>
+
+    <hr class="seperator">
+
+    <section class="section-2">
+      <h2>See the past TechSplendor Symposia</h2>
+      <hr>
+      <p class="evidence">Take a look at the past two TechSplendor Symposia which took place in Gulu and Soroti
+      </p>
+      <div class="about-cards">
+        <div class="about about-1">
+          <p class="yr">2022</p>
+          <p class="venue">TechSplendor Symposim in Gulu</p>
+        </div>
+        <div class="about about-2">
+          <p class="yr">2021</p>
+          <p class="venue">TechSplendor Symposim in Soroti</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="partners partners-about">
+      <h2>Partners</h2>
+      <ul>
+        <li class="partner-1">
+          <a href="">
+            <img src="./images/partners/partner-1.png" alt="Logo of first partner">
+          </a>
+        </li>
+        <li class="partner-2">
+          <a href="">
+            <img src="./images/partners/partner-2.png" alt="Logo of second partner">
+          </a>
+        </li>
+        <li class="partner-3">
+          <a href="">
+            <img src="./images/partners/partner-3.png" alt="Logo of third partner">
+          </a>
+        </li>
+        <li class="partner-4">
+          <a href="">
+            <img src="./images/partners/partner-4.png" alt="Logo of fourth partner">
+          </a>
+        </li>
+        <li class="partner-5">
+          <a href="">
+            <img src="./images/partners/partner-5.png" alt="Logo of fifth partner">
+          </a>
+        </li>
+      </ul>
+    </section>
+  </main>
+
+  <footer class="about-footer">
+    <img src="./images/tech-splendor.png" Footer logo">
+    <div>
+      <small>
+        <p>2015 Creative Commons Korea. Some Rights Reserved.</p>
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly add required `red filter` to two `pictures` and give correct `color` to `text of footer` 🙏🏼 

![image](https://github.com/dd-obua/Mic-Portal-Captone/assets/63915024/97e45da7-ab75-4c0e-baea-f3802bd12d0e)